graylog2-web-interface/src/components/users/UserDetails/UserDetails.jsx

```diff
@@ -28,8 +29,8 @@ const UserDetails = ({ user }: Props) => {
           <SettingsSection user={user} />
         </div>
         <div>
-          <SectionComponent title="Teams">Children</SectionComponent>
           <RolesSection user={user} />
+          <TeamsSection user={user} />
```

Comment:
Could the `user` be added to a Context Provider so that we do not have to pass it to every component?

Reply:
I will discuss this with @linuspahl on monday

Reply:
The advantages are not huge, but I agree we could something like a `UserDetailsContext`.